### ISSUE TITLE
Error in console 

### ISSUE THREAD

**Issue by 4iloveg** [epicmaxco/vuestic-admin#113]
![image](https://user-images.githubusercontent.com/19300333/37707003-8399ecea-2d12-11e8-9fd3-968cb59ca1ca.png)

browser - google chrome  v65.0.3325.181

**Comment by smartapant** [comment]
Hi @4iloveg ! Thanks for reporting!

Can't reproduce - could you please describe how this happened? :)

**Comment by 4iloveg** [comment]
I saw this error on the local site. And then went to your demo and the error is the same, I realized that it was in the browser.
In firefox there is no such error (some kind of message about fonts)
![image](https://user-images.githubusercontent.com/19300333/38022871-6b1d2e08-3289-11e8-8a1d-38d9cf3a5f99.png)

A friend on ubuntu in Google Chrome does not have this error. At me Windows 10.

This link shows that the author of the question found a solution.
https://github.com/wnr/element-resize-detector/issues/81
